src/CloudFoundryApiClient/src/Models/LoginInfoResponse.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Tanzu.Toolkit.CloudFoundryApiClient.Models
+{
```

Comment:
Do you need the sample here?  Would just the link suffice?

Reply:
Yeah idk about this one - I provided both the sample & the link in case the link changes (not sure how likely that is). I know it looks a bit cluttered but I kinda like the sample being inline here since the structure of the `prompts` item wasn't very intuitive to me